toolchain/lowering/lowering_handle.cpp

```diff
@@ -156,6 +156,12 @@ auto LoweringHandleIntegerLiteral(LoweringFunctionContext& context,
   context.SetLocal(node_id, v);
 }
 
+auto LoweringHandleIndex(LoweringFunctionContext& /*context*/,
```

Comment:
Please keep this lexically sorted, between FunctionDeclaration and IntegerLiteral.

I know this is a nit, but please fix this everywhere in this change.

Reply:
You seem to have only fixed this file, can you please fix other files too?

Reply:
This was marked as resolved again, but I'm not seeing changes for this: I'm marking the other locations I'm seeing.